src/components/SubCategorySummary/SubCategorySummary.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import IndicatorCard from '../Card/IndicatorCard/IndicatorCard';
+import aggregateIndicatorForCities from '../../helpers/aggregateIndicatorForCities';
+import getSubCategorySectionId from '../../helpers/getSubCategorySectionId';
+import { INDICATORS } from '../../constants';
+import style from './SubCategorySummary.scss';
+import COLORS from '../../style/_colors.scss';
+
+const SubCategorySummary = (props) => {
+  const backgroundColor = COLORS[`${props.colorName.toUpperCase()}_${props.colorVariation}`];
+
+  const firstThreeIndicators = props.indicatorIds.slice(0, 3);
+
+  const indicatorCardWrappers = firstThreeIndicators.map((indicatorId, i) => {
+    const lastOne = i === (firstThreeIndicators.length - 1);
+
+    return (
+      <div
+        key={indicatorId}
+        className={style.cardWrapper}
+      >
+        <IndicatorCard
+          className={style.card}
+          indicator={indicatorId}
+          value={aggregateIndicatorForCities(
+            indicatorId,
+            props.cities,
+          )}
+        />
+        {lastOne && (
+          <a
+            className={style.linkWrapper}
+            href={`#${getSubCategorySectionId(props.name)}`}
```

Comment:
This is just a straight jump to the section, we can do a fancy slide later if we have the time